projects/packages/forms/src/dashboard/inbox/export-modal/google-drive.js

```diff
@@ -51,6 +53,13 @@ const GoogleDriveExport = ( { onExport } ) => {
 			} );
 	}, [ onExport ] );
 
+	const handleConnectClick = useCallback( () => {
+		tracks.recordEvent( 'jetpack_forms_upsell_googledrive_click', {
+			screen: 'form-responses-inbox',
+		} );
+		pollForConnection();
```

Comment:
Would it make sense to do the `pollForConnection()` call before the tracks record Event? 
This way if the tracks one fails for some reason we still continue with `pollForConnection()`

Reply:
Just checked this, and the event fires fine if I move it after the pollForConnection() call. I don't feel strongly here, so happy to move it after if we'd prefer that.

Reply:
Moved the tracking call after pollForConnection()....